src/content/hi/fundamentals/getting-started/principles/index.markdown

```diff
@@ -1,12 +1,12 @@
 ---
-नक़्शा: साझ/संकुचित पृष्ठों की सूची
-शीर्षक: "साइट डिजाइन के सिद्धांत"
-वर्णन: इसका उद्देश था इस प्रश्न का उत्तर निकालना, 'क्या है जो एक अच्छी मोबाइल साइट बनता है ?'
-पर प्रकाशित: २०१४-०८-०८
-को अपडेट किया: २०१५-०९-१७  
-अनुक्रम: ५ 
-अनुवाद प्राथमिकता : ०
-लेखक:
+layout: साझ/संकुचित पृष्ठों की सूची
+title: "साइट डिजाइन के सिद्धांत"
+description: इसका उद्देश था इस प्रश्न का उत्तर निकालना, 'क्या है जो एक अच्छी मोबाइल साइट बनता है ?'
+published_on: २०१४-०८-०८
```

Comment:
Dates, order and translation priority should not be translated, they're data used by the build process.